lib/growthbook/util.rb

```diff
@@ -39,20 +42,20 @@ def self.checkRule(actual, op, desired)
       end
     end
 
-    def self.chooseVariation(userId, experiment)
-      testId = experiment.id
-      weights = experiment.getScaledWeights
+    def self.choose_variation_for_user(user_id, experiment)
```

Comment:
There's already a `choose_variation` so I needed to call this one something else to avoid a naming conflict.

Reply:
This got cleaned up when deleting the deprecated classes. 🤷‍♀️